pkg/download/automation/automation.go

```diff
@@ -101,6 +105,41 @@ func (d *Downloader) Download(projectName string, automationTypes ...config.Auto
 type NoopAutomationDownloader struct {
 }
 
+func createTemplateFromRawJSON(obj automationClient.Response) (t template.Template, extractedName string) {
+	configId := obj.Id
+
+	var data map[string]interface{}
+	err := json.Unmarshal(obj.Data, &data)
+	if err != nil {
+		log.Warn("Failed to sanitize downloaded JSON for config %v - template may need manual cleanup: %v", configId, err)
```

Comment:
nitpick: Might be beneficial to show what type the config has

Reply:
Hmm.. I think on download it would be most helpful to tell the user the actual filepath - but I'm not sure we can fully do this at this point. I'll take a look though

Reply:
added the type